extensions/extension_types.yaml

```diff
@@ -27,7 +32,11 @@ types:
   - name: multipolygon
     structure:
       T: List<polygon>
+  - name: mixed
```

Comment:
Mixed could apply to any type.  Perhaps make this the any_geometry or any_simple_geometry type?  Fortunately this is internal only so folks shouldn't need to reference it if it has a complicated name.